common/styles/utilities/_root-scope-classes.scss

```diff
@@ -403,18 +403,20 @@
   max-width: 100%;
 }
 
-// These classes should probably become something generic
-// that we can look at when we look at having more breakout type content.
-.person-image {
-  max-width: 100px;
+// Images
+.width-inherit {
+  width: inherit;
 }
 
-.half-in-image {
-  margin-top: -50px;
-}
+.avatar {
```

Comment:
I think I would sooner have this in it's own partial.

Reply:
> partial

❓